requirements_dev.txt

```diff
@@ -1,8 +1,9 @@
+pytest==5.3.1
+tox==3.14.1
+cookiecutter>=1.6.0
+pytest-cookies==0.5.1
+alabaster==0.7.12
+watchdog==0.9.0
```

Comment:
Oh la pinning of dependencies.  We'll have keep up-to-date with upstream often to get newer versions.